cardano/base/templates/home.html

```diff
@@ -7,30 +7,30 @@
       <div class="panel-body">
         <div class="row text-center">
           <div class="col-md-6">
+            {{ mozillians.vouched }}
+            <span data-toggle="tooltip" data-placement="right" class="glyphicon
             {% if vouched_diff > 0 %}
-              <span class="glyphicon glyphicon-triangle-top" aria-hidden="true" title="{{ vouched_diff }}">
-              </span>
+              glyphicon-triangle-top
             {% elif vouched_diff == 0 %}
-              <span class="glyphicon glyphicon-minus" aria-hidden="true" title="{{ vouched_diff }}">
-              </span>
+              glyphicon-minus
             {% else %}
-              <span class="glyphicon glyphicon-triangle-bottom" aria-hidden="true" title="{{ vouched_diff }}">
-              </span>
+              glyphicon-triangle-bottom
             {% endif %}
-            {{ mozillians.vouched }}
+              " aria-hidden="true" title="{{ vouched_diff }}">
+            </span>
           </div>
           <div class="col-md-6">
-              <span class="glyphicon
-              {% if total_diff > 0 %}
-                glyphicon-triangle-top
-              {% elif total_diff == 0 %}
-                glyphicon-minus
-              {% else %}
-                glyphicon-triangle-bottom
-              {% endif %}
-                " aria-hidden="true" title="{{ total_diff }}">
-              </span>
             {{ mozillians.total }}
+            <span data-toggle="tooltip" data-placement="right" class="glyphicon
+            {% if total_diff > 0 %}
```

Comment:
Instead of ``if/elif/else`` and mangling inside ``<span>`` I would have created a helper to get the ``glyphicon`` string.